src/Mustache.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Bugcache;
+
+use Mustache_Engine;
+
+class Mustache {
```

Comment:
Also prefer such "bridge" classes to wrap external libs so the app doesnt depend on them all over the place